testsuite/features/secondary/trad_openscap_audit.feature

```diff
@@ -23,8 +28,9 @@ Feature: openSCAP audit of traditional client
     Then I should see a "Details of XCCDF Scan" text
     And I should see a "Default" text
     And I should see a "XCCDF Rule Results" text
-    And I should see a "pass" text or "notapplicable" text
-    And I should see a "rule-" link
+    When I enter "pass" as the filtered XCCDF result type
+    And I click on the filter button
+    Then I should see a "rule-pwd-warnage" link
```

Comment:
Unless "rule-pwd-warnage" text is visible on the UI and for the user, I'd prefer to have a step definition hiding this technical detail.

Reply:
It is visible on UI and therefore easy to check when we get screenshots.